README.md

```diff
@@ -63,6 +72,11 @@ Ever wanted to share a code snippet effortlessly? Select the code, right-click t
 
 ![](https://storage.googleapis.com/pieces_multimedia/PROMOTIONAL/PIECES_FOR_DEVELOPERS/SUBLIME/share.gif)
 
+### Auto Complete
+
+Auto-completion of code via snippets to speed up your coding process, you can turn this settings off from your settings.
```

Comment:
You can mention that if you have saved Pieces Snippets you get automatic completion via Pieces Snippets. For specific language you can refer to: https://docs.pieces.app/features/code-completion